### ISSUE TITLE
accuracy calculation

### ISSUE THREAD

**Issue by ruanshihai** [CompVis/adaptive-style-transfer#4]
> `x=(pred > tf.ones_like(pred)*0.5)`
I think this is an error as `pred` is not the output of sigmoid.

**Comment by dimakot55** [comment]
@ruanshihai you're right, there is a minor mistake. I believe that it doesn't influence the training process significantly since variable `pred` is logit and takes values in (-inf; +inf). Besides, new models were trained with correct loss and So the correct solution will be `x=(pred > tf.zeros_like(pred))`. Have fixed the error. 
Thanks for mentioning it.
